app/src/outcomes-binary.js

```diff
@@ -18,14 +18,15 @@ const OutcomesBinary = ({ probabilities, stagedProbabilities, isResolved }) => {
     stagedProbability = stagedProbabilities[0];
     predictedProbabilityDifference = stagedProbability.sub(probability);
     absPredictedProbabilityDifference = predictedProbabilityDifference.abs();
-    displayPredictionProbability =
-      absPredictedProbabilityDifference.gte("0.0001") && !isResolved;
+    displayPredictionProbability = absPredictedProbabilityDifference.gte(
+      "0.0001"
```

Comment:
I think this should be a constant somewhere, in the past it caused us problems if we had different "minimal considered values" in different places

Reply:
I think I created that originally, but it would still be nice if you could add that 🙏